### DISCUSSION TITLE
vorta asking for a password

### DISCUSSION THREAD

**Discussion by paolobenve** [borgbase/vorta#1572] [FAQ]
I'm using vorta to automate my borg backup on ubuntu linux. I didn't set any password, but when I manually launch a backup, a window opens saying:  "The application 'vorta-repo' has requested to open the wallet 'kdewallet'. Please enter the password for this wallet below".

I suspect that this password request is the reason why I haven't any automatic backup in my backup archives list.

How do I disable the password request?

**Comment by real-yfprojects** [comment]
Currently the password request cannot be disabled. However you shouldn't run unencrypted backups from a security perspective. 

> I suspect that this password request is the reason why I haven't any automatic backup in my backup archives list.

This might be the reason. In the schedule tab, does vorta display a scheduled time?

**Comment by paolobenve** [comment]
> Currently the password request cannot be disabled

I disabled the "Store repository passwords in system keychain, if available." option, and now backups are run!

Actually I'm running vorta as root, I run it in a terminal as `sudo vorta &`.

`vorta` isn't launched automatically at boot, despite the corresponding option set. Is there a way to fix it?

**Reply by real-yfprojects** [reply]
> `vorta` isn't launched automatically at boot, despite the corresponding option set. Is there a way to fix it?

You can manually add to your systems autostart. Is there a `vorta.desktop` file in `~/.config/autostart`?

**Reply by paolobenve** [reply]
> You can manually add to your systems autostart

does that works for running vorta as root?

**Reply by real-yfprojects** [reply]
> does that works for running vorta as root?

I think, but I can't tell you how.
